tests/http_apis/run.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $cur/../_utils/test_prepare
+
+WORK_DIR=$TEST_DIR/$TEST_NAME
+
+function prepare_data() {
+    run_sql 'DROP DATABASE if exists http_apis;' $MYSQL_PORT1
+    run_sql 'CREATE DATABASE http_apis;' $MYSQL_PORT1
+    run_sql "CREATE TABLE http_apis.t(i TINYINT, j INT UNIQUE KEY);" $MYSQL_PORT1
+    for j in $(seq 100); do
+        run_sql "INSERT INTO http_apis.t VALUES ($j,${j}000$j),($j,${j}001$j);" $MYSQL_PORT1
+    done
+}
+
+function run() {
+    prepare_data
+
+    run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+    check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
+    run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
+    check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT
+
+    echo "start task"
+    curl -X POST 127.0.0.1:$MASTER_PORT/apis/alpha/tasks -d '{"task": "name: test\ntask-mode: all\nis-sharding: false\nmeta-schema: \"dm_meta\"\nremove-meta: false\nenable-heartbeat: true\ntimezone: \"Asia/Shanghai\"\n\ntarget-database:\n  host: \"127.0.0.1\"\n  port: 4000\n  user: \"root\"\n  password: \"\"\n\nmysql-instances:\n  - source-id: \"mysql-replica-01\"\n    black-white-list:  \"instance\"\n    mydumper-config-name: \"global\"\n    loader-config-name: \"global\"\n    syncer-config-name: \"global\"\n\nblack-white-list:\n  instance:\n    do-dbs: [\"http_apis\"]\n\nmydumpers:\n  global:\n    mydumper-path: \"./bin/mydumper\"\n    threads: 4\n    chunk-filesize: 0\n    skip-tz-utc: true\n    extra-args: \"-B http_apis --statement-size=100\"\n\nloaders:\n  global:\n    pool-size: 16\n    dir: \"./dumped_data\"\n\nsyncers:\n  global:\n    worker-count: 16\n    batch: 100\n    max-retry: 100"}' > $WORK_DIR/start-task.log
```

Comment:
is it better to post the task config from a local file?

Reply:
https://github.com/pingcap/dm/pull/224/commits/e8a64f1035010bd04bb7dcba1b15964385c5b79f

Reply:
is it possible to use curl like this `curl -F "file=@/path/to/task/config"`, and don't need to escape `\"` in task config file, so we can use the same task config file from dmctl and http API

Reply:
seems -F used to upload file, I can't find a better way. and I removed escape in task.yaml in https://github.com/pingcap/dm/pull/224/commits/da366529ea570c357dbf8341f0dee367f5b03ed6